File: ios/Sources/ApplePayPlugin/ApplePay.swift
import Foundation

@objc public class ApplePay: NSObject {
    @objc public func echo(_ value: String) -> String {
        print(value)
        return value
    }
    
    @objc public func canMakePayments(_ value: Bool) -> Bool {
        print(value)
        return value
    }

    @objc public func showApplePaySheet(_ value: Bool) -> Bool {
        print("showApplePaySheet")
        return value
    }

    @objc public func showInAppPurchaseSheet(_ value: Bool) -> Bool {
        print("showInAppPurchaseSheet")
        return value
    }
}

File: android/src/main/java/com/paulmojicatech/ApplePay.java
package com.paulmojicatech;

import android.util.Log;

public class ApplePay {

    public String echo(String value) {
        Log.i("Echo", value);
        return value;
    }

    public Boolean canMakePayments() {
        Log.i("canMakePayments", "false");
        return false;
    }
}
